doc/documentation.rst

```diff
@@ -431,6 +431,8 @@ There are also **examples**, which contain a short use-case to highlight MNE-fun
     auto_examples/inverse/plot_mne_crosstalk_function.rst
     auto_examples/inverse/plot_mne_point_spread_function.rst
     auto_examples/inverse/plot_morph_data.rst
+    auto_examples/inverse/plot_morph_stc_to_fsaverage_volume.rst
+    auto_examples/inverse/plot_morph_VolSourceEstimate.rst
```

Comment:
plot_morph_VolSourceEstimate -> plot_morph_vol_stc

never put camel case / capital letters in file names